### DISCUSSION TITLE
Contributor call: 5 March 2022

### DISCUSSION THREAD

**Discussion by darigovresearch** [freesewing/freesewing#1910] [Support] [answered]
You are all invited to the FreeSewing Contributor Call.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This issue will be updated after the call with notes and to-dos.

### About
FreeSewing's contributor call is an informal hangout/chat/meeting that is open to all FreeSewing contributors and users alike.
The call is organized/hosted by @tangerineshark and @lucibytes on our Discord server.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

Details:

Location: https://discord.freesewing.org (meeting room: 🔊voice chat)
Date: 6 March 2022 4pm UTC, which is can be found in your local time zone [here](https://savvytime.com/converter/utc-to-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/Mar-5-2022/5pm)
Or check the bot in Discord! #meetings

### Agenda
- To be determined

> This discussion will be updated after the call with notes and to-dos.

**Comment by l3xander** [comment, marked as answer]
There was no agenda! But we did cover some things :) 

- @joostdecock streamed the new .org site lay-out, showing new features that make it easier to cut and sort patterns.
    - @joostdecock is working on an overview where you can see how your pattern will be printed, in whatever paper size you choose, with the ability to move individual pattern parts around to save paper.
    - He’s also working on doing this for fabric, that you can say “my fabric is measured X by Y” and you can see how you’d most efficiently place the pattern to save the most fabric while taking the grainline into account. But the focus for now is the previous feature.
    - There are some things that make this a pain to implement though, there are pesky bugs and we also know this feature won’t be accepted by our current tiler. It has to be figured out how to get it to PDF.
- The newsletter deadline is creeping closer! It will come out on April 1st.
    - We’re throwing an announcement out on the #community Discord channel to see if anyone would like to write anything.
    - No big features have come out lately, mainly “behind the scenes” work done. Though @bobgeorgethe3rd and @/wren emphasized that these updates are fun to read too and that it’s good to let people know that work *is* being done.
    - @tangerineshark mentioned that we have @eleonoremilbou’s interview and we can make a second interview if we need more content.
    - @darigovresearch mentioned we could write something about @bobgeorgethe3rd’s bucket hat, even though it isn’t finished yet.
    - @tangerineshark also mentioned they could write an article on how to make ‘non-breast’ patterns into patterns that you can use as a busted person.
    - The next contributor call is the last call where we will be able to talk about the newsletter, so let’s keep that in mind.

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions
